ftw/zipextract/profiles/default/metadata.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<metadata>
+    <dependencies>
+        <dependency>profile-ftw.upgrade:default</dependency>
```

Comment:
`ftw.zipexport` should not install `ftw.upgrade`.
For declaring upgrade steps it is not necessary to install the generic setup profile of `ftw.upgrade`; so it is the job for policy packages / projects to install it.